README.md

```diff
@@ -164,8 +173,8 @@ you can change polyfill’s encoder:
 
 ```diff
   window.MediaRecorder = require('audio-recorder-polyfill')
-+ MediaRecorder.encoder = require('./ogg-opus-encoder')
-+ MediaRecorder.mimeType = 'audio/ogg'
++ MediaRecorder.encoder = require('audio-recorder-polyfill/ogg-opus-encoder')
```

Comment:
We do not have `audio-recorder-polyfill/ogg-opus-encoder`, let’s use old value

Reply:
actually, this is the old old value :) but fine

Reply:
Old value was `./ogg-opus-encoder` where `.` is a developer’s project

Reply:
OCD FTW :)